From e0222c8ad6c7c285b875fe727d9823f7d42235cb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Wed, 12 Nov 2025 20:16:58 +0100
Subject: [PATCH 056/133] cmake: update ZIG_LLVM_REQUIRED_TARGETS to contain
 LoongArch and SPIRV
X-Developer-Signature: v=1; a=openpgp-sha256; l=924; i=xtex@astrafall.org;
 h=from:subject; bh=MJckyDGCutcmYes1hL2cX/7vxEXP2j2++7999yklBiw=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn78hj33fsdt7+v50hWqfailL3OrjLcHx/5yuUnPrt3r
 sb5oplLx0QWBjEuBksxRZYiwwZvVp10ftFl5bIwc1iZQIZIizQwMDAwsDDw5SbmlRrpGOmZahvq
 GRrqGOkYMXBxCsBUz/zE8Fcg3E6u440Oj9+RF8Fs2tWfjvltn9vY8+K9QLHPpo1GDnMZGb6Et+c
 4n/Q2b7WZc3P7jZoU320VLy8524n81mHIXbbAlgcA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit be2a902784dd26199ada0409a5d32e9f5443752a)
---
 cmake/Findllvm.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/Findllvm.cmake b/cmake/Findllvm.cmake
index 5ff5de869c93..068d04d7d3b8 100644
--- a/cmake/Findllvm.cmake
+++ b/cmake/Findllvm.cmake
@@ -83,7 +83,7 @@ if(ZIG_USE_LLVM_CONFIG)
       OUTPUT_STRIP_TRAILING_WHITESPACE)
     string(REPLACE " " ";" LLVM_TARGETS_BUILT "${LLVM_TARGETS_BUILT_SPACES}")
 
-    set(ZIG_LLVM_REQUIRED_TARGETS "AArch64;AMDGPU;ARM;AVR;BPF;Hexagon;Lanai;Mips;MSP430;NVPTX;PowerPC;RISCV;Sparc;SystemZ;VE;WebAssembly;X86;XCore")
+    set(ZIG_LLVM_REQUIRED_TARGETS "AArch64;AMDGPU;ARM;AVR;BPF;Hexagon;Lanai;LoongArch;Mips;MSP430;NVPTX;PowerPC;RISCV;SPIRV;Sparc;SystemZ;VE;WebAssembly;X86;XCore")
 
     set(ZIG_LLVM_REQUIRED_TARGETS_ENABLED TRUE)
     foreach(TARGET_NAME IN LISTS ZIG_LLVM_REQUIRED_TARGETS)
-- 
2.52.0

